UIPS/UIP-0103.md

```diff
@@ -51,14 +51,35 @@ Syntax for this hint could be a modified version of the `~+` rune that includes
 ~+  /ford/parse
 ```
 
+There will be three categories of possible paths:
+1. `/`: the empty path, which targets the transient cache.
```

Comment:
```
> !=  ~+(=== 1)  ::  clue doesn't always expand to [1 0]
[11 [1.869.440.365 1 3] 1 1]
```

The triple `tis` is implemented in Hoon but not used currently. We need to handle it. Just add a check to see if the clue is a `[1 @]`. If it is, stick it in the transient cache.